activerecord/test/schema/schema.rb

```diff
@@ -1023,6 +1023,7 @@
     t.integer :horrible_polymorphic_man_id
     t.string  :horrible_polymorphic_man_type
     t.references :human, polymorphic: true, index: false
+    t.integer :autosave_man_id
```

Comment:
```suggestion
```

I don't think we need to add a new column here, the autosave associations can reuse the same foreign key.